### DISCUSSION TITLE
Add box shadow to the latest release

### DISCUSSION THREAD

**Discussion by ayushsharma82** [community/community#6497] [Other Feature Feedback, Questions, & Ideas] [answered]
Adding a light box-shadow to the latest release card will signify that it's the latest one on a first glance. The scrolling nature for large boxes currently makes the UI a bit confusing.

Just like this shadow:
![image](https://user-images.githubusercontent.com/26592968/137806486-1ca1c76b-8d18-4bbe-b339-a2f9d31b9e11.png)

**Comment by MylesBorins** [comment, marked as answer]
@ayushsharma82 we attempted to follow the overall design system of GitHub.com when creating the release cards. As such adding a drop shadow just for the release cards would be inconsistent with the design of the rest of github.com. As such were are unfortunately not going to be able to do anything with this request, but I do appreciate you opening the discussion!
